<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegistrationForm</class>
 <widget class="QDialog" name="RegistrationForm">
  <property name="windowTitle">
   <string>Регистрация происшествий</string>
  </property>
  <layout class="QVBoxLayout" name="mainLayout">
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <!-- Состояние №1: Основные данные -->
     <widget class="QWidget" name="state1">
      <layout class="QFormLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="labelIncidentType">
         <property name="text"><string>Тип происшествия:</string></property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="comboIncidentType">
         <item><property name="text"><string>Бедствие</string></property></item>
         <item><property name="text"><string>Авария</string></property></item>
         <item><property name="text"><string>Тренировка</string></property></item>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="labelIncidentDateTime">
         <property name="text"><string>Дата и время происшествия (UTC):</string></property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QDateTimeEdit" name="dateTimeIncident">
         <property name="displayFormat"><string>dd.MM.yyyy HH:mm</string></property>
         <property name="calendarPopup"><bool>true</bool></property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="labelLocation">
         <property name="text"><string>Местоположение происшествия:</string></property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="editLocation"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="labelCoordinates">
         <property name="text"><string>Координаты (широта, долгота):</string></property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="editCoordinates"/>
       </item>
      </layout>
     </widget>

     <!-- Состояние №2: Дополнительные сведения -->
     <widget class="QWidget" name="state2">
      <layout class="QFormLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="labelVesselName">
         <property name="text"><string>Название судна:</string></property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="editVesselName"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="labelIMO">
         <property name="text"><string>IMO номер:</string></property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="editIMONumber"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="labelCallSign">
         <property name="text"><string>Позывной:</string></property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="editCallSign"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="labelPersons">
         <property name="text"><string>Людей на борту:</string></property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QSpinBox" name="spinPersonsOnBoard"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="labelSituation">
         <property name="text"><string>Описание ситуации:</string></property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QTextEdit" name="textSituationDescription"/>
       </item>
      </layout>
     </widget>

     <!-- Состояние №3: Подтверждение -->
     <widget class="QWidget" name="state3">
      <layout class="QVBoxLayout">
       <item>
        <widget class="QPlainTextEdit" name="plainConfirm">
         <property name="readOnly"><bool>true</bool></property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonLayout">
     <item>
      <widget class="QPushButton" name="buttonPrev">
       <property name="text"><string>Назад</string></property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonNext">
       <property name="text"><string>Далее</string></property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonFinish">
       <property name="text"><string>Готово</string></property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>